### PR TITLE
SAC page done and bootstrap 4 cdn linked in index base

### PR DESCRIPTION
**Pull request by heisen-bug**
Still need to get sac images in sac.html. Right now I have added random pics, but need to update as soon as we get SAC images

### CODE REVIEW

File: swd/templates/indexbase.html [hunk 1]
```diff
@@ -7,6 +7,10 @@
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
   <meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
 
+  <!-- Bootstrap CDN -->
```
Comment (aryanagarwal15) [inline swd/templates/indexbase.html]: Materialize.css already being used. No need for bootstrap. Please use materialize.css only.

File: swd/main/views.py [hunk 2]
```diff
@@ -73,14 +73,14 @@ def login_success(request):
 #     url = Student.objects.get(user=request.user).profile_picture
 #     print(url)
 #     ext = url.name.split('.')[-1]
-    
+
```
Comment (aryanagarwal15) [inline swd/main/views.py]: White spaces will not be counted as code changed. Please remove all white space changes.

File: swd/main/views.py [hunk 3]
```diff
@@ -331,7 +331,7 @@ def messoption(request):
             total_amount += tee.totamt
     for other in otherdues:
         if other is not None:
-            total_amount += other.amount            
+            total_amount += other.amount
```
Comment (aryanagarwal15) [inline swd/main/views.py]: No, white space changes will be counted as code changed, please remove all white space changes

File: swd/templates/sac.html [hunk 4]
```diff
@@ -1,127 +1,77 @@
...
-				<th style="text-align: center">Incharge</th>
-				<th </th>
-			</tr>
+    <div class="left-content">
+        <h5 class='head'>Student Activity Center (SAC)</h5>
+        <div class="upper-part">
+            <div class="row">
+                <div class="col-lg-4 left-part">
+                    <h6 class='table-head'>Incharge</h6>
+                    <hr width='30px' color='#ff8000' align='left' style='height: 5px;padding: 0;margin: 0 margin-bottom: 10px'>
```
Comment (aryanagarwal15) [inline swd/templates/sac.html]: All styles should be used either in a single file or as in file style attribute, try not to use inline style attributes.

**Comment by aryanagarwal15** [comment]
This is the link to the mobile prototype: https://drive.google.com/open?id=1o0Y560MGhWWp7vaDsT5D7CNY4Pmv1VjM

Please try and make the whole thing responsive as well. Please use materialize 0.100.2 for the project and not bootstrap. (Link to Docs: http://archives.materializecss.com/0.100.2/)

**Comment by aryanagarwal15** [comment]
#211

**Comment by aryanagarwal15** [comment]
This is solving #211

**Comment by Mr-Codemeister** [comment]
Recreated everything using materialcss and did the setup for mobile view. Have a look.

**Pull request merged**